masonry_core/src/passes/accessibility.rs

```diff
@@ -120,9 +129,6 @@ fn build_access_node(
     if ctx.is_disabled() {
         node.set_disabled();
     }
-    if ctx.is_stashed() {
```

Comment:
Is it worthwhile to potentially have a debug assert here that isn't stashed just to verify that in debug builds?